articles/java-jdbc/index.md

```diff
@@ -0,0 +1,309 @@
+                    .prepareStatement("insert into  students values (?, ?, ?)");
+            preparedStatement.setString(1, student.getName());
+            preparedStatement.setString(2, student.getCourse());
+            preparedStatement.setString(3, student.getEmail());
+            preparedStatement.executeUpdate();
+        } catch (Exception e) {
+            System.out.println(e.getLocalizedMessage());
+        }
+
+    }
+
+    public void deleteStudent(String email) {
+        try {
+            preparedStatement = connect
+                    .prepareStatement("delete from school.students where email= ? ; ");
+            preparedStatement.setString(1, email);
+            preparedStatement.executeUpdate();
+
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+
+        }
+
+    }
+
+    public void updateStudent(Student student, String email) {
+        try {
+            preparedStatement = connect
+                    .prepareStatement("update students set name= ? where email= ? ;");
+            preparedStatement.setString(1, student.getName());
+            preparedStatement.setString(2, email);
+            preparedStatement.executeUpdate();
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void close() {
+        try {
+            if (resultSet != null) {
+                resultSet.close();
+            }
+
+            if (statement != null) {
+                statement.close();
+            }
+
+            if (connect != null) {
+                connect.close();
+            }
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+}
+```
+
+- `databaseConnection()` method establishes a connection to the database. `connect = DriverManager.getConnection("jdbc:mysql://localhost/school?user=root&password=");` sets up the database connection information. `school` is the database name, `root` is the database username, and the database password is a blank string ``.
+- `getStudents()` method returns a list of students from the database. ` resultSet = statement.executeQuery("select * from school.students");` executes the `SELECT` query that gets all the students from the database and stores the result in the `resultSet`. 
+- The `while (resultSet.next())` loops through the result set to get all the students' data returned.
+- The `getStudentByEmail(String email)` method returns a student from the database with the email password in the `SELECT` query.
+- The `saveStudent(Student student)` method inserts a new student passed to the method into the database.
+- The `deleteStudent(String email)` method deletes a student with the email passed to the `DELETE` query in the database.
+- This `updateStudent(Student student, String email)` method updates the student information of the student with the email passed to the `UPDATE` query.
+- The `close()` method closes the database connection.
```

Comment:
It will be better to include all the above methods explanation at the begin of the section. For instance, You can say the `MysqlAccess.java` file will contain the following methods
`databaseConnection`
- Method does what
- specific code snippet for that method
`saveStudent`
- Method does what
- specific code snippet for that method

After explaining the methods, you can now provide the entire code for the `MysqlAccess.java file`. 
I believe these changes will align better with the topic `Getting started with Java JDBC`.